Dangerfile

```diff
@@ -4,7 +4,7 @@
 #      and: https://github.com/samdmarshall/pyconfig/blob/develop/Dangerfile
 
 # set the number of lines that must be changed before this classifies as a "Big PR"
-@S_BIG_PR_LINES ||= 500
+@S_BIG_PR_LINES ||= 0
```

Comment:
I think this is valid imo. I like this, otherwise we'll go back to never updating changelog.